pymc_marketing/constants.py

```diff
@@ -12,3 +12,4 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 DAYS_IN_YEAR: float = 365.25
+DAYS_IN_MONTH: float = DAYS_IN_YEAR / 12
```

Comment:
These are what are used as the periods of `YearlyFourier` and `MonthlyFourier`
Do you think they should be changed? @juanitorduz

Reply:
Nah, let's leave it like this for now